src/cpp/session/include/session/SessionOptions.hpp

```diff
@@ -309,6 +309,16 @@ class Options : boost::noncopyable
       return allowOverlay() || allowRpubsPublish_;
    }
 
+   bool allowExternalPublish() const
```

Comment:
We should update these docs as well (this is on rstudio-pro): https://github.com/rstudio/rstudio-pro/blob/master/doc/articles/05-r-sessions/06-feature-limits.md